src/v/cluster/tests/partition_balancer_simulator_test.cc

```diff
@@ -919,3 +920,44 @@ FIXTURE_TEST(test_replica_pair_frequency, partition_balancer_sim_fixture) {
     validate_even_replica_distribution();
     validate_replica_pair_frequencies();
 }
+
+FIXTURE_TEST(test_mixed_replication_factors, partition_balancer_sim_fixture) {
+    add_node(model::node_id{0}, 100_GiB, 1);
+    add_node(model::node_id{1}, 400_GiB, 1);
+    add_node(model::node_id{2}, 100_GiB, 1);
+
+    // Corner case from real deployment with the following configuration of
```

Comment:
Uh, it was better before clang-format :) Maybe add `*` before each "partitions: ", so that it doesn't try to format the list?

Reply:
you can also use `// clang-format: off ... // clang-format: on` or something like that.

Reply:
i tried with `*`, i think the only way is to go with `clang-format: off`